index.html

```diff
@@ -1,569 +1,476 @@
+                            fill="currentColor" />
+                    </svg>
+                </a>
+            </div>
         </div>
 
-        <ul class="languages">
-          <li class="language">
-            <img src="assets/images/image5.png" alt="" />
-            <p>javascript</p>
-          </li>
-
-          <li class="language">
-            <img src="assets/images/image6.png" alt="" />
-            <p>html</p>
-          </li>
-
-          <li class="language">
-            <img src="assets/images/css7.png" alt="" />
-            <p>css</p>
-          </li>
-        </ul>
-
-        <div class="tools-dropdown">
-          <div class="tool-dropdown">
-            <p>Frameworks</p>
-            <i class="bx bx-chevron-right"></i>
-          </div>
-
-          <div class="tool-dropdown">
-            <p>Frameworks</p>
-            <i class="bx bx-chevron-right"></i>
-          </div>
+        <div class="tools-summery">
+            <div class="header">
+                <h2>languages</h2>
+                <i class='bx bx-chevron-down'></i>
+            </div>
+
+
+            <ul class="languages">
+                <li class="language">
+                    <img src="assets/images/image5.png" alt="">
+                    <p>javascript</p>
+                </li>
+
+                <li class="language">
+                    <img src="assets/images/image6.png" alt="">
+                    <p>html</p>
+                </li>
+
+                <li class="language">
+                    <img src="assets/images/css7.png" alt="">
+                    <p>css</p>
+                </li>
+            </ul>
+
+            <div class="tools-dropdown">
+                <div class="tool-dropdown">
+                    <p>Frameworks</p>
+                    <i class='bx bx-chevron-right'></i>
+                </div>
+
+                <div class="tool-dropdown">
+                    <p>Frameworks</p>
+                    <i class='bx bx-chevron-right'></i>
+                </div>
+            </div>
```

Comment:
- [ ] **Required:** You have the 'Frameworks' text repeated, when it must be 'Frameworks' and 'Skills', please update it as my colleague also mentioned